images/common/scripts/complement_jail.sh

```diff
@@ -110,6 +110,9 @@ pushd "${jaildir}"
         flock etc/complement_jail_setcap_enroot_aufs2ovlfs.lock -c "setcap cap_sys_admin,cap_mknod+pe usr/bin/enroot-aufs2ovlfs"
     fi
 
+    echo "Shared folder for pyxis output /var/cache/enroot-container-images"
```

Comment:
Do we need such folder on login node? If not you can move that part to the worker side (`if [ -z "$worker" ]`)

Reply:
I would keep it, so in case you want to see what images you have, you don't need to ssh to worker, you can just simply list it from login node